src/app/+item-page/item-page.resolver.ts

```diff
@@ -28,6 +28,7 @@ export class ItemPageResolver implements Resolve<RemoteData<Item>> {
       followLink('owningCollection'),
       followLink('bundles'),
       followLink('relationships'),
+      followLink('version', undefined, followLink('versionhistory')),
```

Comment:
I'm getting an error on this line when I try to run this PR with AoT compilation (`yarn start`) or via `yarn run watch`.  Here's the error:
```
ERROR in app/+item-page/item-page.resolver.ts:31:40 - error TS2345: Argument of type 'FollowLinkConfig<HALResource>' is not assignable to parameter of type 'boolean'.

31       followLink('version', undefined, followLink('versionhistory')),
```

Reply:
@tdonohue This is due to #588 that was just merged. I'll let @Atmire-Kristof know